modules/candidate_parameters/jsx/participantStatus.js

```diff
@@ -0,0 +1,295 @@
+      var suboptionsRequired = false;
+      for (var key in required) {
+        if (required.hasOwnProperty(key)) {
+          var participantStatus = this.state.formData.participant_status;
+          if (participantStatus === null || participantStatus === undefined) {
+            participantStatus = this.state.Data.participant_status;
+          }
+          if (required[key].ID === participantStatus) {
+            subOptions = this.state.Data.parentIDs[participantStatus];
+            suboptionsRequired = true;
+            break;
+          }
+        }
+      }
+
+      var formattedHistory = [];
+      for (var statusKey in this.state.Data.history) {
+        if (this.state.Data.history.hasOwnProperty(statusKey)) {
+          var line = "";
+          for (var field in this.state.Data.history[statusKey]) {
+            if (this.state.Data.history[statusKey].hasOwnProperty(field)) {
+              var current = this.state.Data.history[statusKey][field];
+              if (current !== null) {
+                switch (field) {
+                  case 'data_entry_date':
+                    line += "[";
+                    line += current;
+                    line += "] ";
+                    break;
+                  case 'entry_staff':
+                    line += current;
+                    line += " ";
+                    break;
+                  case 'status':
+                    line += " Status: ";
+                    line += current;
+                    line += " ";
+                    break;
+                  case 'suboption':
+                    line += "Details: ";
+                    line += current;
+                    line += " ";
+                    break;
+                  case 'reason_specify':
+                    line += "Comments: ";
+                    line += current;
+                    line += " ";
+                    break;
+                  default:
+                }
+              }
+            }
+          }
+          formattedHistory.push(<p>{line}</p>);
+        }
+      }
+
+      var alertMessage = "";
+      var alertClass = "alert text-center hide";
+      if (this.state.updateResult) {
+        if (this.state.updateResult === "success") {
+          alertClass = "alert alert-success text-center";
+          alertMessage = "Update Successful!";
+        } else if (this.state.updateResult === "error") {
+          var errorMessage = this.state.errorMessage;
+          alertClass = "alert alert-danger text-center";
+          alertMessage = errorMessage ? errorMessage : "Failed to update!";
+        }
+      }
+
+      return (
+            <div>
+                <div className ={alertClass} role="alert" ref="alert-message">
+                    {alertMessage}
+                </div>
+            <FormElement
+                name ="participantStatus"
+                onSubmit ={this.handleSubmit}
+                ref ="form"
+                class ="col-md-6"
```

Comment:
When you are adding a column class to an element, make sure that it (the element) is wrapped in another element with a `row` class to maintain correct layout. In this case the root `<div>` should have class `row`.